tests/model/model_controller/DefaultModelController.cc

```diff
@@ -260,3 +265,72 @@ TEST_F(DefaultModelControllerTest, GetSpecificSolidTasks) {
     EXPECT_EQ(result.AccessResult()[i], expected[i]);
   }
 }
+
+TEST_F(DefaultModelControllerTest, MustCallSaveOnPersistence) {
+  auto _0 = task_factory_.GetNextTask();     // NOLINT
+  auto __1 = task_factory_.GetNextTask();    // NOLINT
+  auto ___3 = task_factory_.GetNextTask();   // NOLINT
+  auto ____7 = task_factory_.GetNextTask();  // NOLINT
+  auto ___6 = task_factory_.GetNextTask();   // NOLINT
+  auto __2 = task_factory_.GetNextTask();    // NOLINT
+  auto _4 = task_factory_.GetNextTask();     // NOLINT
+  auto __5 = task_factory_.GetNextTask();    // NOLINT
+  auto _8 = task_factory_.GetNextTask();     // NOLINT
+  model_controller_->Add(_0);
+  model_controller_->Add(CreateTaskId(0), __1);
+  model_controller_->Add(CreateTaskId(0), __2);
+  model_controller_->Add(CreateTaskId(1), ___3);
+  model_controller_->Add(_4);
+  model_controller_->Add(CreateTaskId(4), __5);
+  model_controller_->Add(CreateTaskId(1), ___6);
+  model_controller_->Add(CreateTaskId(3), ____7);
+  model_controller_->Add(_8);
+
+  auto expected = model_controller_->GetAllSolidTasks().AccessResult();
+
+  EXPECT_CALL(*persistence_, Save(expected)).Times(1);
+  model_controller_->Save();
```

Comment:
could fetch and check the corresponding OperationResult

Reply:
Fixed